core/object/script_language.h

```diff
@@ -363,6 +363,7 @@ class ScriptLanguage : public Object {
 
 	virtual Error lookup_code(const String &p_code, const String &p_symbol, const String &p_path, Object *p_owner, LookupResult &r_result) { return ERR_UNAVAILABLE; }
 
+	virtual String format_code(const String &p_code) { return p_code; }
```

Comment:
We might want to add this to `ScriptLanguageExtension` as well.

Reply:
I think I've added what's necessary - please check.

Reply:
Looks fine at a first glance. I'm not really familiar with the whole virtual binding system, but I think it might be necessary to use `GDVIRTUAL_CALL` instead of `GDVIRTUAL_REQUIRED_CALL` to not break compatibility :thinking:

Reply:
Done